test/end-to-end/core.sh

```diff
@@ -405,7 +405,10 @@ os::cmd::try_until_text 'oc get pods -l openshift.io/deployer-pod.type=hook-post
 os::cmd::expect_success 'oc create -f test/testdata/failing-dc.yaml'
 os::cmd::try_until_success 'oc get rc/failing-dc-1'
 os::cmd::expect_failure 'oc rollout status dc/failing-dc'
-os::cmd::expect_success_and_text 'oc logs dc/failing-dc' 'test pre hook executed'
+# FIXME: getting the hook logs seems to be flaking, re-enable when we find the
```

Comment:
Please don't do it like this :( we need the test, we just need to slightly modify it so that docker has a chance to get that last line. Add `sleep 1` like [here](https://github.com/openshift/origin/blob/d3cdfecb089dc67721400a8d7df26b723cd305a9/test/extended/testdata/deployments/custom-deployment.yaml#L29) in `test/testdata/failing-dc.yaml`rather than disabling this test.

/hold

Reply:
please don't do `sleep 1` that just makes the flake less common, not fixed.

Reply:
agree with @tnozicka adding `sleep 1` will just hide this problem and we will never get to fixing it.

Reply:
We (openshift team) don't own docker and we won't fix it, we can't block our team just to know about this bug. There should be an open issue to remove those sleeps but for now we need to limit the # of flakes.